src/_components/blurredModal/BlurredModal.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FC, PropsWithChildren } from 'react';
+import { Modal } from 'react-native';
+
+import * as Styled from './style';
+
+type TLogOutModalProps = {
+  isVisible: boolean;
+  toggleIsVisible: (nextValue?: unknown) => void;
```

Comment:
`nextValue` isn't passed in anywhere, so can be removed

```suggestion
  toggleIsVisible: () => void;
```